llvm/test/CodeGen/SPIRV/extensions/SPV_INTEL_joint_matrix/cooperative_matrix_bf16.ll

```diff
@@ -0,0 +1,46 @@
+; RUN: llc -O0 -mtriple=spirv64-unknown-unknown --spirv-ext=+SPV_KHR_cooperative_matrix,+SPV_INTEL_joint_matrix %s -o - | FileCheck %s
```

Comment:
If it's something that spirv-val understands, we need to add also smth like

; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv64-unknown-unknown --spirv-ext=+SPV_KHR_cooperative_matrix,+SPV_INTEL_joint_matrix %s -o - -filetype=obj | spirv-val %}

in all new test cases.

Also -verify-machineinstrs if it doesn't fail.

Reply:
spirv-val won't understand it as the capabilities and the instructions are not yet propagated to SPIR-V Headers.